### PR TITLE
add templates for domains

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature? |  yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Bugfixes tag also because i fixed the redirection when created an item from template (we were redirect to the template instead the new created item)

### CODE REVIEW

File: inc/commondbtm.class.php [hunk 1]
```diff
@@ -1067,7 +1067,9 @@ function add(array $input, $options = [], $history = true) {
             $id_to_clone = $input['id'];
          }
          if (isset($id_to_clone) && $this->getFromDB($id_to_clone)) {
-            return $this->clone($input, $history);
+            $new_id = $this->clone($input, $history);
+            $this->getFromDB($new_id);
+            return $new_id;
```
Comment (cedric-anne) [inline inc/commondbtm.class.php:1072]: I think this piece of code should be applied on 9.5/bugfixes branch too.

Comment (cedric-anne) [reply inc/commondbtm.class.php:1072]: See #7777.

File: inc/log.class.php [hunk 2]
```diff
@@ -79,17 +79,13 @@ static function getTypeName($nb = 0) {
...
-         }
-         return self::createTabEntry(self::getTypeName(1), $nb);
+      $nb = 0;
+      if ($_SESSION['glpishow_count_on_tabs']) {
+         $nb = countElementsInTable('glpi_logs',
+                                    ['itemtype' => $item->getType(),
+                                       'items_id' => $item->getID()]);
       }
-      return '';
+      return self::createTabEntry(self::getTypeName(1), $nb);
```
Comment (cedric-anne) [inline inc/log.class.php:88]: Should this be changed in 9.5/bugfixes branch ?

Comment (orthagh) [reply inc/log.class.php:88]: as seen together, yes

Comment (cedric-anne) [reply inc/log.class.php:88]: Done in #7784.

**Pull request merged**